database/schema.sql

```diff
@@ -4,7 +4,11 @@ DROP TABLE IF EXISTS parentOrgs;
 CREATE TABLE parentOrgs (
   id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
   name VARCHAR UNIQUE NOT NULL,
-  logo VARCHAR
+  logo VARCHAR,
+  Description VARCHAR DEFAULT '',
+  skills VARCHAR [],
+  term VARCHAR,
```

Comment:
An organization would come in multiple terms and years. How can you store it as a string?

Reply:
By term I meant the latest term "spring" "Fall" etc